src/auth/tests.js

```diff
@@ -11,6 +12,28 @@ let cookie;
 app.get('/protected', requireAuth, (req, res) => res.sendStatus(200));
 
 describe('Auth api', () => {
+  before(async () => {
+    const usersData = [
+      {
+        email: 'admin@babajka.io',
+        password: 'password',
+      },
+    ];
+    await Promise.all(
+      usersData.map(async (userData) => {
+        const user = new User(userData);
+        await user.setPassword(userData.password);
+        return user.save();
+      })
+    );
+  });
+
+  after(async () => {
+    await Promise.all([
+      User.remove({ email: 'admin@babajka.io' }),
```

Comment:
if this is generic solution for future using, make sense to use here emails from `usersData`, exclude copy-pasting, or just to create / remove only one user.

Reply:
`usersData` is something we're using when running locally, not for unit-testing. I'd better keep it like that. Renamed the user so that it's not overlapped with users from `usersData`.